packages/terra-framework-docs/src/terra-dev-site/test/pills/filter-pills/CollapsibleFilterPills.test.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import classNames from 'classnames/bind';
+import FilterPills from '@cerner/terra-pills/lib/index';
+import styles from './FilterPillsTestCommon.module.scss';
+
+const cx = classNames.bind(styles);
+
+const CollapsibleFilterPills = () => {
+  const pillsData = [
+    {
+      label: 'asthma',
+      id: 'terra-filter-pills-pill-asthma',
+    },
+    {
+      label: 'bronchitis',
+      id: 'terra-filter-pills-bronchitis',
+    },
+    {
+      label: 'fibro',
+      id: 'terra-filter-pills-fibro',
+    },
+    {
+      label: 'allergies',
+      id: 'terra-filter-pills-allergies',
+    },
+    {
+      label: 'hypertension',
+      id: 'terra-filter-pills-hypertension',
+    },
+    {
+      label: 'cardiac',
+      id: 'terra-filter-pills-cardiac',
+    },
+  ];
+  const [pills, setPills] = useState(pillsData);
+  const [counter, setCounter] = useState(0);
+
+  const handleOnRemove = (pillKey, metaData) => {
+    const pillsArray = pills;
+    pillsArray.splice(metaData.index, 1);
+    setPills([...pillsArray]);
+  };
+
+  const handleAddPills = () => {
+    const oldPills = pills;
+    const pill = {
+      label: `test-pill- ${counter}`,
+      id: `test-filter-pills-test-${counter}`,
+    };
+    setCounter(newCounter => newCounter + 1);
+    oldPills.push(pill);
+    setPills(oldPills);
+  };
+
+  return (
+    <>
+      <FilterPills
+        ariaLabel="Example of collapsed filter pills"
+        onRemove={handleOnRemove}
+        isCollapsible
+        className={cx(['container', 'show-border', 'width-50'])}
+      >
+        {pills.map((pill, index) => (
+          <FilterPills.Pill
+            label={pill.label}
+            id={pill.id}
+            key={pill.id}
```

Comment:
What is the reason for setting the `key` prop here?

Reply:
FilterPills container has a role of "list". Each item within this container expects a unique `key`, else we get a console warning.